src/preact/utils/README.md

```diff
@@ -0,0 +1,37 @@
+# Bento Services
+
+Most of the AMP Classic components depend on the various services in the `src/service` folder.
+
+Bento components, however, cannot rely on those services.
+
+So this folder contains standalone versions of these services, designed to make it easier to convert Classic components
+to Bento components.
+
+For example:
+
+Using AMP Classic services:
+
+```js
+import { Services } from '#service';
+///...
+
+const platform = Services.platformFor(this.win);
+if (platform.isIos()) {
+  // do Apple stuff
+} else if (platform.isAndroid()) {
+  // do Android stuff
+}
+```
+
+Using Bento services:
+
+```js
+import { platformUtils } from "#preact/services/platform";
```

Comment:
Nit: I personally prefer exporting the functions directly and allowing consuming modules to `import * as platformUtils` or `import {isIos} from '#preact/services/platform'` as appropriate. It's cleaner, easier to refactor, and the extra object seems like an unnecessary abstraction considering we're most (all?) of the services we're converting are singleton classes

Reply:
I totally agree, I really prefer exporting individual functions instead of util objects.
However, we've been looking at ways to support document-level scoping for most of these utils, so these will be re-written very soon. We are tracking this work separately from this PR: https://github.com/ampproject/amphtml/issues/37465

Reply:
I'd thought that they got tree-shaken off (similar to how `import * as platformUtils` does but looks like [it does not](https://rollupjs.org/repl/?version=2.66.1&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMiUyRiolMjBUUkVFLVNIQUtJTkclMjAqJTJGJTVDbmltcG9ydCUyMCU3QiUyMG1hdGhVdGlscyUyMCU3RCUyMGZyb20lMjAnLiUyRm1hdGhzLmpzJyUzQiU1Q24lNUNuY29uc29sZS5sb2coJTIwbWF0aFV0aWxzLmN1YmUoJTIwNSUyMCklMjApJTNCJTIwJTJGJTJGJTIwMTI1JTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlN0QlMkMlN0IlMjJuYW1lJTIyJTNBJTIybWF0aHMuanMlMjIlMkMlMjJjb2RlJTIyJTNBJTIyJTJGJTJGJTIwbWF0aHMuanMlNUNuJTVDbiUyRiUyRiUyMFRoaXMlMjBmdW5jdGlvbiUyMGlzbid0JTIwdXNlZCUyMGFueXdoZXJlJTJDJTIwc28lNUNuJTJGJTJGJTIwUm9sbHVwJTIwZXhjbHVkZXMlMjBpdCUyMGZyb20lMjB0aGUlMjBidW5kbGUuLi4lNUNuZnVuY3Rpb24lMjBzcXVhcmUlMjAoJTIweCUyMCklMjAlN0IlNUNuJTVDdHJldHVybiUyMHglMjAqJTIweCUzQiU1Q24lN0QlNUNuJTVDbiUyRiUyRiUyMFRoaXMlMjBmdW5jdGlvbiUyMGdldHMlMjBpbmNsdWRlZCU1Q25mdW5jdGlvbiUyMGN1YmUlMjAoJTIweCUyMCklMjAlN0IlNUNuJTVDdCUyRiUyRiUyMHJld3JpdGUlMjB0aGlzJTIwYXMlMjAlNjBzcXVhcmUoJTIweCUyMCklMjAqJTIweCU2MCU1Q24lNUN0JTJGJTJGJTIwYW5kJTIwc2VlJTIwd2hhdCUyMGhhcHBlbnMhJTVDbiU1Q3RyZXR1cm4lMjB4JTIwKiUyMHglMjAqJTIweCUzQiU1Q24lN0QlNUNuJTVDbmV4cG9ydCUyMGNvbnN0JTIwbWF0aFV0aWxzJTIwJTNEJTIwJTdCJTVDbiU1Q3RzcXVhcmUlMkMlMjBjdWJlJTIwJTVDbiU3RCUyMiUyQyUyMmlzRW50cnklMjIlM0FmYWxzZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmFtZCUyMiUzQSU3QiUyMmlkJTIyJTNBJTIyJTIyJTdEJTJDJTIyZ2xvYmFscyUyMiUzQSU3QiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE)